sources/packages/web/src/services/http/ApplicationApi.ts

```diff
@@ -133,4 +135,34 @@ export class ApplicationApi extends HttpBaseClient {
       throw error;
     }
   }
+
+  /**
+   * API Client for application detail.
+   * @param applicationId
+   * @param userId
+   * @returns
+   */
+  public async getApplicationDetails(
+    applicationId: number,
+    studentId: number,
+  ): Promise<GetApplicationBaseDTO> {
+    const response = await this.getCall(
+      `application/aest/${applicationId}/student/${studentId}`,
```

Comment:
shouldn't it be ` aest/application/${applicationId}/student/${studentId}`,

Reply:
As discussed, moving aest to the end of URI